packages/amplify-provider-awscloudformation/src/utils/auth-types.ts

```diff
@@ -9,7 +29,7 @@ export interface AdminAuthPayload {
   region: string;
 }
 
-export interface AuthConfig extends AdminAuthPayload {
+export interface AdminAuthConfig extends AdminAuthPayload {
```

Comment:
I think it's better to define things like this using composition instead of inheritance:
```ts
export type AdminAuthConfig = AdminAuthPayload & {IdentityId: string}
```